ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/HARouting.java

```diff
@@ -462,4 +473,6 @@ private static void validateSchema(
     SUCCESS,
     STANDBY_FALLBACK
   }
+
+  private static class Alan1 { }
```

Comment:
😆 I'm tempted to keep this as an easter egg

Reply:
oops.  Removed!